src/rtc_base/platform_thread_types.h

```diff
@@ -22,6 +22,8 @@
 #elif defined(WEBRTC_FUCHSIA)
 #include <zircon/types.h>
 #include <zircon/process.h>
+#elif defined(WEBRTC_OPENBSD)
+#include <pthread.h>
```

Comment:
Why do you need a separate condition here? The WEBRTC_POSIX path include it as well